javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,96 @@
+//#1 WEATHER APP:
+
+//#a Sign up for api key:(Done!)
+
+//#b First call to the weather api:(Done!)
+
+//#c Fetch weather data from a city:
+const notification = document.querySelector("#notification");
+const button = document.querySelector("#submit");
+const inputValue = document.querySelector("#input-value");
+const cityName = document.querySelector("#city");
+const temperature = document.querySelector("#temp");
+const icon = document.querySelector("#weather-icon");
+const wind = document.querySelector("#wind-speed");
+const cloud = document.querySelector("#cloudiness");
+const sunrise = document.querySelector("#sunrise");
+const sunset = document.querySelector("#sunset");
+//const map = document.querySelector("#Map");
+
+//Weather App Data
+const weather = {};
+weather.temperature = {
+  unit: "celcius",
+};
+
+//App Const:
+const KELVIN = 237;
+
+//Checking if browser supports geolocation:
+// if ("geolocation" in navigator) {
+//   navigator.geolocation.getCurrentPosition(setPosition, showError);
+// } else {
+//   notification.style.display = "block";
+//   notification.innerHTML = "Browser does not support geolocation";
+// }
+//Setting User's position:
+// function setPosition(position) {
+//   const longitude = position.coords.longitude;
+//   const latitude = position.coords.latitude;
+// }
+// getWeather(longitude, latitude);
+
+//Show message when there is an Error with geolocation:
+function showError(error) {
+  notification.style.display = "block";
+  notification.innerHTML = `${error.message}`;
+}
+
+// Converting the Unix_timestamp:
+function convertTimestamp(Unix_timestamp) {
+  const date = new Date(Unix_timestamp * 1000);
+  const hour = date.getHours();
+  const minutes = "0" + date.getMinutes();
+  const seconds = "0" + date.getSeconds();
+  return hour + ":" + minutes.substr(-2) + ":" + seconds.substr(-2);
+}
+
+//convert temperature to Fahrenheit/celcius:
+function celciusToFahrenheit(temperature) {
+  return (temperature * 9) / 5 + 32;
+}
+celciusToFahrenheit();
```

Comment:
you're not using the return value. Nothing will happen with the number you got from the function.